### ISSUE TITLE
[SODA] Text alternative for alert image is missing when creating a collection

### ISSUE THREAD

**Issue by aaron-collier** [sul-dlss/hungry-hungry-hippo#908]
The text alternative for the informative image is missing. When one of the alert images is present, this information needs to be conveyed to all users by including a text equivalent.

Soda recommendation:
```
RULE : 
Images that convey content MUST have programmatically-discernible alternative text.

HOW TO FIX: 
Fix this issue by using ONE of the following techniques:

1. Use the alt attribute on the <img> element to convey the intent, purpose, or meaning of the image.

<img src="sunset.jpg" alt="The sun setting over the Pacific Ocean, with a silhouette of a flying seagull in the foreground">

<img src="cat.svg" alt="cat" role="img"> (role is optional, for backwards compatibility with Safari)

2. Use aria-label and role="img" to convey the intent, purpose, or meaning of the image.

<p>Have a great day <span class="fonticon smiley" role="img" aria-label="Smiley face"></span></p>

<svg role="img" aria-label="cat">

3. Use CSS clipped text to provide screen reader accessible (non-visible) alternative text.

<a href="https://www.facebook.com/dequesystems" class="fb">
<span class="sr-text">Deque's Facebook page</span>
</a>

.sr-text {
position: absolute;
clip: rect(1px, 1px, 1px, 1px);
-webkit-clip-path: inset(50%) ;
clip-path: inset(50%);
padding: 0;
border: 0;
height: 1px;
width: 1px;
white-space: nowrap;
overflow: hidden;
}

REFERENCE: 
Deque University: https://dequeuniversity.com/class/images/alt-text/informative
W3C-WAI tutorial: https://www.w3.org/WAI/tutorials/images/

BACKGROUND: 
People who are blind cannot see images on a page. In order to give people who cannot see an image access to the information conveyed by the image, it must have a text alternative. The text alternative must describe the information or function represented by the image. Screen readers can then use the alternative text to convey that information to the screen reader user.

For informative images - images that convey information - the alternative text must communicate the intent, purpose, or meaning of the image in a way that serves as a true alternative for the image.

It is not possible to provide alternative text by adding an alt attribute directly to CSS background images. Images that convey content should not generally be placed in CSS. However, if it is necessary to do so, you can add alternative text to informative and actionable CSS images using other methods.
```

See issue id 2258 in https://docs.google.com/spreadsheets/d/1x0uBBDe3T8O0HNCs74OdVKQc8tQQNvOV_f-C6IS44IQ/edit?gid=1050712151#gid=1050712151

**Comment by astridu** [comment]
I'm not sure, but I'm guessing they're talking about the little red alert icons that show up in the left nav since there is no other error text around it. I think we can resolve this by either putting alt text that says "Error alert" OR somehow indicating that tab has an error. Is there an aria label we can use to signify when the tab is in error? That might be more useful (e.g. "Title and contact has an error").

**Comment by aaron-collier** [comment]
@justinlittman I agree with @astridu here, I'm pretty sure they are talking about the icon used in the tab, possibly also the icon in the alert. They say this in the report:

```
It is not possible to provide alternative text by adding an alt attribute directly to CSS background images. Images that convey content should not generally be placed in CSS. However, if it is necessary to do so, you can add alternative text to informative and actionable CSS images using other methods.
````

I believe the issue they are reporting is that we are setting the content via css (https://github.com/sul-dlss/hungry-hungry-hippo/blob/main/app/assets/stylesheets/application.bootstrap.scss#L93) AND that specific bit is being used to convey content. We should be able to use one of their suggestions to update this, I'm working on an example now and we can discuss if necessary.
